notebooks/nessie-iceberg-demo-nba.ipynb

```diff
@@ -0,0 +1,654 @@
+{
```

Comment:
I'm confused. Is this the same as https://github.com/projectnessie/nessie-demos/pull/144/files#diff-309382582ac7fa7e43ebfad2a4cb3a2c1cbbacff7e218bc8046304d5ff286382 ?
Needs a rebase?

Reply:
yes, rebased now. Thats why it was still in draft